doc/src/index.rst

```diff
@@ -14,20 +15,67 @@ If you are new to SymPy, start with the :ref:`Tutorial <tutorial>`.
 
 This is the central page for all of SymPy's documentation.
 
+==================================================================
+
+..  rst-class:: column
+
+:ref:`Getting started <installation>`
+-------------------------------------------------------
+
+An introduction to SymPy for developers and detailed tutorials.
```

Comment:
This is not really accurate. The section as it is now is just installation instructions.